src/org/loklak/api/aggregation/ClassifierServlet.java

```diff
@@ -0,0 +1,174 @@
+package org.loklak.api.aggregation;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.loklak.data.DAO;
+import org.loklak.http.RemoteAccess;
+import org.loklak.server.Query;
+
+
+public class ClassifierServlet extends HttpServlet {
```

Comment:
@vibhcool: I have implemented it using `AbstractAPIHandler`. Please take a look again.

> `/abc/classifier` looks better than `/abc/classifier.json`.

I was just sticking to the earlier followed conventions.